src/context/errors.ts

```diff
@@ -6,7 +6,9 @@ import { json } from './json'
  * Returns a list of GraphQL errors.
  */
 export const errors = (
-  errorsList: Partial<GraphQLError>[],
-): ResponseTransformer<{ errors: typeof errorsList }> => {
+  errorsList?: Partial<GraphQLError>[],
+): ResponseTransformer | ResponseTransformer<{ errors: typeof errorsList }> => {
+  if (!errorsList) return json({})
```

Comment:
That will set the response body to an empty object and may not play nicely when combined with `ctx.data`, as it may override any preceding data. 

Can we return the response transformer without any transformations in this case?
```js
if (!errorsList) {
  return res => res
}
```

> We may later replace this with something like `identity()` function for a shorter notation.

Reply:
make sense 👍